packages/amplication-client/src/style/mixin.scss

```diff
@@ -125,6 +125,12 @@ $bullet-size: 6px;
   align-items: center;
 }
 
+@mixin flexColumn {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
```

Comment:
please make sure this one is required, or do we already have a solution for this use case?

Reply:
I searched for these CSS properties before I added this mixin. We don't have something like that, and I assume this is a part of the new design of the sidebars

Reply:
ok